editor/script/script_text_editor.cpp

```diff
@@ -2185,12 +2192,53 @@ bool ScriptTextEditor::can_drop_data_fw(const Point2 &p_point, const Variant &p_
 					String(d["type"]) == "nodes" ||
 					String(d["type"]) == "obj_property" ||
 					String(d["type"]) == "files_and_dirs")) {
+		set_drop_info_text(d);
 		return true;
 	}
 
 	return false;
 }
 
+void ScriptTextEditor::set_drop_info_text(const Dictionary &p_info) const {
+	String text;
+
+	static const String single_resource_text = TTR("Drag and drop to copy the file path into the script.") +
+			"\n" + TTR("Hold Ctrl when dropping to add a const var, preloading the resource at that path.") +
```

Comment:
This should use `keycode_get_string((Key)KeyModifierMask::CMD_OR_CTRL)` to be correct for mac platform (same for all mentions of Ctrl).